pkg/printer/githubactions.go

```diff
@@ -9,22 +9,32 @@ import (
 )
 
 // GitHubActions is a GitHubActions printer meant for use by a GitHub Action annotation
-type GitHubActions struct{}
+type GitHubActions struct{ writer io.Writer }
```

Comment:
Fixed. I've seen a few weird things with `golangci-lint` that I would have thought it would catch but it didn't. For example, the uses of `print.Print()` in the code should be complaining about not checking the return error but it's not. Overall it works pretty great though, so *shrug*.